src/test/java/biz/paluch/logging/gelf/GelfDynamicMdcFieldTypeUnitTests.java

```diff
@@ -0,0 +1,83 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one or more
```

Comment:
Please use the copyright header from existing files with its formatting. Let's not re-format license headers.